### PR TITLE
cadc-vos data structure API improvement

### PR DESCRIPTION
**Pull request by pdowler**
change: `public final {collection}...` to `private final {collection} ...` + get method

fixes for above changes
some tweaks to the owner handling in XML reader/writer
started making cadc-vos-client work

### CODE REVIEW

File: cadc-vos-client/src/main/java/org/opencadc/vospace/client/Main.java [hunk 1]
```diff
@@ -615,29 +611,25 @@ private String getContentLength(Node n, boolean simple) {
...
-        }
-
-        View view = null;
-        if (originalDestination != null) {
-            view = createAcceptsView(new VOSURI(originalDestination), null);
-        }
-        if (view == null) {
-            view = new View(VOS.VIEW_DEFAULT);
-        }
+        //URI originalDestination = null;
```
Comment (andamian) [inline cadc-vos-client/src/main/java/org/opencadc/vospace/client/Main.java:614]: I think a TODO is missing here.

**Review by andamian — APPROVED** [review]
Not sure if the temporarily commented out code needs TODOs tags to remember to go back or not so I left a few comments on that.

File: cadc-vos-client/src/intTest/java/org/opencadc/vospace/client/VOSpaceClientTest.java [hunk 2]
```diff
@@ -145,7 +146,7 @@ public void setUp() throws Exception {
         //        log.debug("hostname=" + hostname);
         //        endpoint = "https://" + hostname;
         //        log.debug("endpoint=" + endpoint);
-        client = new VOSpaceClient(URI.create("ivo://cadc.nrc.ca/vospace"));
+        //client = new VOSpaceClient(URI.create("ivo://cadc.nrc.ca/vospace"));
```
[the remark this answers is not in the document]
Comment (pdowler) [reply cadc-vos-client/src/intTest/java/org/opencadc/vospace/client/VOSpaceClientTest.java:149]: they mostly do not even compile yet, I was probably starting to do something and then decided to postpone

File: cadc-vos-client/src/main/java/org/opencadc/vospace/client/Main.java (hunk 1 above)
Comment (pdowler) [reply cadc-vos-client/src/main/java/org/opencadc/vospace/client/Main.java:614]: yeah, also just doing enough to get it to work for basics so commenting out extra features... it still needs a lot of work and wasn't even included in the branch CI before

**Pull request merged**